src/commands/install/install_package_from_repo.py

```diff
@@ -0,0 +1,49 @@
+from logging import getLogger
+from os import path
+from typing import Optional
+
+from colorama import Fore
+from git import InvalidGitRepositoryError, NoSuchPathError
+from git.objects import Submodule
+from git.repo import Repo
+
+from src.commands.install import installation_exceptions
+
+
+def install_package_from_repo(
+    name: str,
+    url: str,
+    tag: Optional[str],
+    repo_root_dir: str,
+    destination: str,
+):
+    logger = getLogger()
+
+    try:
+        repo = Repo(repo_root_dir)
+    except InvalidGitRepositoryError as _err:
+        raise installation_exceptions.InvalidLocalRepository()
+    except NoSuchPathError as _err:
+        raise installation_exceptions.InvalidLocalRepository()
```

Comment:
```suggestion
```
Why do we catch this exceptions?

Reply:
It is a foundation for making errors “user-friendly”.  I am planning to create an abstract `ProtostarException` that all custom exceptions in the app should extend. In this way, we can later determine whether to display a nice error message, e.g. “Git repository is not initialized. Please run `git init`.” or  yeet stack trace.

Reply:
I'll add handling validation error in this task: https://github.com/software-mansion/protostar/issues/33.